docs/overview_and_guide.rst

```diff
@@ -126,6 +126,7 @@ In order to use Raiden correctly and safely there are some things that need to b
 - **Persistency of local DB**: Your local state database is located at ``~/.raiden``. This data should not be deleted by the user or tampered with in any way. Frequent backups are also recommended. Deleting this directory could mean losing funds.
 - **Raiden Always online**: Make sure that your node is always working, your network connection is stable and that the Raiden node is always online. If it crashes for whatever reason you are responsible to restart it and keep it always online. We recommend running it inside some form of monitor that will restart if for some reason the raiden node crashes.
 - **Ethereum Client Always Online**: Make sure that your ethereum client is always running and is synced. We recommend running it inside some form of monitor that will restart if for some reason it crashes.
+- **Ethereum Client can not be changed**: Swapping the ethereum client while transactions are not mined is considered unsafe. We recommend avoiding switching ethereum clients once the Raiden node is running.
```

Comment:
What do you mean by "while transactions are not mined"?

Reply:
The transaction is in the transaction pool but it's not yet in a mined block.

Reply:
Can you formulate it like that instead? I think that's more clear than your first suggestion

Reply:
But that's the definition of "transaction is not mined"

Reply:
So, should I rephrase it or not?

If yes, suggestion:

> - **Ethereum Client can not be changed**: Swapping the ethereum client while transactions are in the pool and not yet included in a block is considered unsafe. We recommend avoiding switching ethereum clients once the Raiden node is running.